server/src/api/customer/customer.model.ts

```diff
@@ -5,6 +5,7 @@ export default class Customer extends Model {
   id!: number;
   roadmapId!: number;
   name!: string;
+  email!: string;
```

Comment:
```suggestion
  email!: string | null;
```

Kenttä on nullable tietokannassa, joten tämä olisi hyvä näkyä myös tyypissä